utils/address.ts

```diff
@@ -0,0 +1,6 @@
+export const shortenAddress = (address: string) => {
+  if (address.length < 9) {
+    return address;
+  }
+  return `${address.slice(0, 4)}...${address.slice(-5)}`;
```

Comment:
Because all addresses have a `0z` prefix we probably should show the first 4 and last 4 characters, *excluding* the prefix. So `address.slice(0, 6)}...${address.slice(-4)

Reply:
Actually I think the prefix is `0zk`, so we'll need `(0,7)` as the first slice

Reply:
This should be fixed now